osi_trafficsign.proto

```diff
@@ -418,7 +418,7 @@ message TrafficSign
                 //
                 TYPE_SNOW_OR_ICE = 94;
```

Comment:
DANGER is missing here.
I'd propose to handle this consistently and thuss remove  DANGER from all types